extensions/amp-analytics/0.1/visibility-model.js

```diff
@@ -348,9 +353,17 @@ export class VisibilityModel {
   isVisibilityMatch_(visibility) {
     dev().assert(visibility >= 0 && visibility <= 1,
         'invalid visibility value: %s', visibility);
+    // Special case: If visiblePercentageMin is 100%, then it doesn't make
+    // sense to do the usual (min, max] since that would never be true.
     if (this.spec_.visiblePercentageMin == 1) {
       return visibility == 1;
     }
+    // Special case: If visiblePercentageMin and Max are both 0, then we
+    // want to ping when the creative becomes not visible.
+    if (this.spec_.visiblePercentageMin == 0 &&
```

Comment:
for consistency, like how we ignored visiblePercentageMax in line 358, we can ignore visiblePercentageMin here as well. (or we can add visiblePercentageMax above)

Reply:
Ok, that consistency sounds good.

Reply:
Done